### DISCUSSION TITLE
Trying to get the module show up

### DISCUSSION THREAD

**Discussion by rowild** [wellenplan/wellenplan#43] [Q&A] [answered]
I am new to directus and directus extension development. So I thought I look at your extensions. But I encounter some problem.

On my local instance, I install `npm install @wellenplan/directus-extension-hourclock-module`. Then I moved the content of `dist/` to `[directus-root]/extensions/modules/hourclock/index.js`. Now I would expect to see a new module in settings, that needs to be activated.

But instead I see nothing. 

How do I have to install your extension?

**Comment by hairmare** [comment, marked as answer]
a plain old `npm install @wellenplan/directus-extension-hourclock-module` should be more than enough to install an extension. you don't need to move the installation anywhere since directus discovers the extension installed via npm using [this](https://github.com/directus/directus/blob/40f13f14bb1a937a50d3277afeaa25102e22f832/packages/shared/src/constants/extensions.ts#L22) regex.

btw, the hourclock-module extension is my least polished one, for something that is less confusing when not used for wellenplan, i'd recommend checking out the duration extensions.

**Reply by rowild** [reply]
Thank you very much for your fast reply and the additional infos, @hairmare ! Very much appreciated!

I was told on Discord that I had to move the ext to their folders manually. Maybe I misunderstood or maybe I formulated my question wrong, but it is good to know that Directus has an "auto detect" mode!

I chose o look at this extension, because the question came: how can the left menu be populated with links. So far it seems this ext is the only "module" out there that can be found on npmjs.com. Therefore my investigation.

I will, of course, look at all your other extensions, and this will not have been my last question, I assume :-)

Thanks again! Have a good day!
